.github/workflows/prerelease_linux.yml

```diff
@@ -141,7 +142,7 @@ jobs:
           username: ${{ env.DOCKER_HUB_ID }}
           password: ${{ env.DOCKER_HUB_PASSWORD }}
       - name: Publish ${{ matrix.assetsType }} to S3 action
-        uses: newrelic/infrastructure-publish-action@v1.0.4
+        uses: newrelic/infrastructure-publish-action@v1.0.7
```

Comment:
why not `v1`?

Reply:
It is better to pin the version, like with all resources the we use. For OHIs we use v1 as there 25+ repos to maintain